src/App.js

```diff
@@ -1,10 +1,176 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import Button from './components/Button';
 
 function App() {
+  const [aarbeien, setAarbeien] = useState(0);
+  const [bananen, setBananen] = useState(0);
+  const [appels, setAppels] = useState(0);
+  const [kiwies, setKiwies] = useState(0);
+
+  function ResetNumbers(){
+    setAarbeien (0);
+    setBananen ( 0);
+    setAppels (0);
+    setKiwies ( 0);
+
+  }
+
+  const [formName, setFormName] = useState('');
+  const [formName2, setFormName2] = useState('');
+  const [formAge, setFormAge] = useState(0);
+  const [formPostCode, setFormPostCode] = useState('');
+  const [formOpmerking, setFormOpmerking] = useState('');
+  const [formBezorgFrequenctie, toggleFormBezorgFrequenctie] = useState('Iedere week');
+  const [formPeriode, toggleFormPeriode] = useState('overdag');
+  const [formNewsletter, toggleFormNewsletter] = useState('false');
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    console.log(formName, formName2, formAge, formPostCode, formOpmerking, formBezorgFrequenctie, formPeriode, formNewsletter, aarbeien, bananen, appels, kiwies);
+  }
+
   return (
     <>
       <h1>Fruitmand bezorgservice</h1>
+        Aarbeien
+        <button type="button" disabled={aarbeien === 0} onClick={() => setAarbeien(aarbeien - 1)}>-</button>
+        <input type="text" value={aarbeien}/>
+        <button type="button" onClick={() => setAarbeien(aarbeien + 1)}>+</button>
```

Comment:
Je had dit in een los component kunnen stoppen, zodat je dat kon hergebruiken voor alle fruitsoorten. Nu zie ik toch wel veel code dubbeling